lib/graphql/static_validation/rules/variable_usages_are_allowed.rb

```diff
@@ -78,6 +78,11 @@ def non_null_levels_match(arg_type, var_type)
     if arg_type.kind.non_null? && !var_type.kind.non_null?
       false
     elsif arg_type.kind.wraps? && var_type.kind.wraps?
+      # If var_type is a non-null wrapper for a type, and arg_type is nullable, peel off the wrapper
+      if var_type.kind == GraphQL::TypeKinds::NON_NULL && arg_type.kind != GraphQL::TypeKinds::NON_NULL
```

Comment:
Can we use `.non_null?` here instead of comparing to those constants? 

I think its a bit better because it matches the surrounding code and it might be easier to change in the future. (No dependency on specific constants, just the call chain `.kind.non_null?`)